ReSharper.FSharp/test/data/parsing/Let - Rec 01.fs

```diff
@@ -1,2 +1,2 @@
-let rec foo x =
+let rec foo (x : string * (int * string)) =
```

Comment:
Please add a separate test case if you want to cover such type usages (if it's not already covered 🤔).

Reply:
Didn't mean to commit this - good spot.